requirements.txt

```diff
@@ -1,2 +1,2 @@
 qiskit-terra>=0.17
-qiskit-ibmq-provider>=0.13
\ No newline at end of file
+qiskit-ibmq-provider
```

Comment:
I assume you changed this to get things working (since 0.13 hasn't been released yet)?

Reply:
Yeah, the issue was `pip install -r requirements.txt` fails because it can't find the version. So I switched to using this without a minimum and then added an install from git in the `tox.ini` for CI. Once 0.13.0 is released we can add the minimum here and remove that line from the `tox.ini`.